bank.rb

```diff
@@ -44,10 +48,10 @@ def withdraw(amount)
       new_balance = @balance - amount
```

Comment:
On line 36 above it seems like you should re-use the `find` method rather than re-writing the same logic.